changelog/3204.feature

```diff
@@ -0,0 +1 @@
+Captured logs are printed before entering pdb.
```

Comment:
This reminds me, please update `1478.feature` as well. 👍

Reply:
Does towncrier reference this PR (#3234) then, or do we have to update the generated changelog manually?

Reply:
It uses the number in the filename to create a link, so the file above will link to #3204